```
ys.path additions: ['/pum/datamodel/app', '/pum/datamodel']
Traceback (most recent call last):
  File "/usr/local/bin/pum", line 6, in <module>
    sys.exit(cli())
             ^^^^^
  File "/usr/local/lib/python3.12/site-packages/pum/cli.py", line 632, in cli
    param = config.parameter(p[0])
            ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/pum/pum_config.py", line 261, in parameter
    raise PumConfigError(f"Parameter '{name}' not found in configuration.") from None
pum.exceptions.PumConfigError: Parameter 'webgis' not found in configuration.

Error: Process completed with exit code 1.

```

--- a/datamodel/.pum.yaml
+++ b/datamodel/.pum.yaml
@@ -45,6 +45,11 @@ parameters:
     description: load ci modification
     app_only: true
 
+  - name: webgis
+    type: boolean
+    default: false
+    description: enable WebGIS app modification
+    app_only: true
 
   - name: modification_yaml
     type: path

--- a/datamodel/app/modification_repos/webgis/config.yaml
+++ b/datamodel/app/modification_repos/webgis/config.yaml
@@ -1,2 +1,1 @@
 sql_files:
-

--- a/datamodel/app/modification_repos/webgis/config.yaml
+++ b/datamodel/app/modification_repos/webgis/config.yaml
@@ -1,0 +1,2 @@
+sql_files:
+

